web/charts/common/lib/src/chart/bar/base_bar_renderer.dart

```diff
@@ -337,7 +335,7 @@ abstract class BaseBarRenderer<D, R extends BaseBarRendererElement,
 
     final orderedSeriesList = getOrderedSeriesList(seriesList);
 
-    orderedSeriesList.forEach((final ImmutableSeries<D> series) {
+    orderedSeriesList.forEach((final series) {
```

Comment:
I can delete it and see if any of the linters complain. I'm more interested to know if @kevmoo is willing to go down this path =)

Reply:
this is copy-pasted from somewhere else. I didn't change anything.

Anything we do here may get thrown away when we remerge. I don't use final in `forEach`. Don't care much either way...

Reply:
I'm guessing this is a copy from https://github.com/google/charts/tree/master/charts_common which itself appears to be sync'ed across from Google3? Can I get a pointer back to the source so I can fix these issues upstream and then just chase it through the waterfall. Seems wrong to fix it here where it will just get thrown away.

Reply:
@kevmoo I think this is the last outstanding question preventing Brett from landing the PR. Any advice?

Reply:
It's fine to land. Just keep in mind we may have to go back through and clean this up again if there is another sync.

Reply:
Landed. Now, how do you feel about Andrew and I making similar changes to the rest of the flutter web samples? =)